development/docker-compose-background.yml

```diff
@@ -13,7 +13,7 @@
 #  the License.
 
 # Docker Compose Spec Version
-version: '2'
+# version: '2'
```

Comment:
I think it makes some sense to have a background compose file to match the behavior of "carma start" where adding no argument to "carma swarm deploy" still launches minimal services. Paul mentioned it's separate since carma-web-ui can launch all the services by itself, though I'm certain that functionality is broken with swarm

Also I probably need to add the registry service in docker-compose-background anyways

Reply:
I modified the carma script to work with both the standard `carma start all` and the Swarm deployment since it may be useful to have an alternative for comparison in integration testing. If we exclusively use Swarm they may be unnecessary, but if we wanted to merge to develop both deployments should work with no conflicts.